controllers/adminController.js

```diff
@@ -25,4 +26,19 @@ exports.getIndex = (req, res, next) => {
                 }
             }
         ) 
+};
+
+exports.postDeletePlayer = (req, res, next) => {
+    console.log('Here DESTROY PLAYER(S)');
+    const emailsArray = req.body.data
+    db.getDb()
+        .db('softball')
+        .collection('players')
+        .deleteMany({ email: {$in: emailsArray } })
+        .then(() => {
+            console.log('DESTROYED PLAYER(S)');
+            console.log(emailsArray)
+            res.redirect('/admin/players');
```

Comment:
What is this `redirect`? This endpoint doesn't "send" anything. We need to send something like `e.g) status(200).json({message: "ok"})`. Server always should give a response in the end.